diffsmhm/tests/testing_catalogs/make_mock_halos.py

```diff
@@ -0,0 +1,195 @@
+import numpy as np
+import h5py
+
+import os
+
+
+def _write_file(fname, hid, upid, pid,
+                mpeak, host_mpeak,
+                x, y, z, hx, hy, hz,
+                host_dist):
+
+    with h5py.File(fname, "w") as f:
+        f.create_dataset("halo_id", data=hid)
+        f.create_dataset("upid", data=upid)
+        f.create_dataset("pid", data=pid)
+
+        f.create_dataset("mpeak", data=mpeak)
+        f.create_dataset("host_mpeak", data=host_mpeak)
+
+        f.create_dataset("x", data=x)
+        f.create_dataset("y", data=y)
+        f.create_dataset("z", data=z)
+
+        f.create_dataset("host_x", data=hx)
+        f.create_dataset("host_y", data=hy)
+        f.create_dataset("host_z", data=hz)
+
+        f.create_dataset("host_dist", data=host_dist)
+
+
+def make_test_catalogs(t):
```

Comment:
Right now this test helper function only returns the input data and not the expected answers. Instead the expected answers are defined in the testing routines themselves. We should return the expected answers with the test data from this function to keep everything in the same spot.